include/swift/AST/Attr.def

```diff
@@ -695,6 +688,12 @@ SIMPLE_DECL_ATTR(_noAllocation, NoAllocation,
   ABIStableToAdd | ABIStableToRemove | APIStableToAdd | APIStableToRemove,
   124)
 
+SIMPLE_DECL_ATTR(_predatesConcurrency, PredatesConcurrency,
+  OnFunc | OnConstructor | OnProtocol | OnGenericType | OnVar | OnSubscript |
+  OnEnumElement | UserInaccessible |
+  ABIBreakingToAdd | ABIBreakingToRemove | APIBreakingToAdd | APIBreakingToRemove,
+  125)
+
```

Comment:
I'm surprised there were no serialization changes to match the addition and removal of these attributes. Is there a semantic break in the serialization format that would justify a version bump?

Reply:
Hmm. We've removed some attributes that were basically unused, but otherwise haven't changed anything. However, we did have one use of `@_unsafeSendable` in the concurrency library, so *in theory* we could run into trouble deserializing an old module because of that. I can separately bump the module format to account for that.